README.md

```diff
@@ -1,5 +1,18 @@
-# license-list-XML
-This is a workspace for developing and implementing a new XML source format for the SPDX License List. This source format is (or will be) used to generate the authoritative file formats for licenses on the SPDX License List. These authoritative [target/destination formats](https://github.com/spdx/license-list-data) will remain the same for the time being.
+# SPDX License List 
+The [SPDX License List](https://spdx.org/licenses/) is a list of commonly found licenses and exceptions used in free and open source and other collaborative software or documentation. The purpose of the SPDX License List is to enable easy and efficient identification of such licenses and exceptions in an SPDX document, in source files or elsewhere. The SPDX License List includes a standardized short identifier, full name, vetted license text including matching guidelines markup as appropriate, and a canonical permanent URL for each license and exception. 
```

Comment:
“software or documentation” → “software and documentation”, because the license list is used for both classes of content.  Also “source files or elsewhere” → “source files and elsewhere” for the same reasons (previous discussion [here][1]).

And I'd prefer “matching guidelines markup” → “matching markup” (or “matching-guidelines markup”) and “canonical permanent URL” → “canonical URL” as I mentioned [here][2].

Now that we're closer to the XML source, I'd also consider using “cross references” for the URL, since the `<crossRefs>` tag doesn't sound link “single, canonical URL” to me.  It's a bit hard to figure out the semantics without docs though (#495).

[1]: https://github.com/spdx/license-list-XML/pull/538#discussion_r158381181
[2]: https://github.com/spdx/tools/pull/133#discussion_r158865454

Reply:
Another possible approach to the first sentence:

> The [SPDX License List](https://spdx.org/licenses/) is a list of licenses and exceptions commonly used in collaborative software and documentation.

That lets us collect the found/used bit in a single place, instead of putting “commonly found” out in front of “licenses and exceptions” while the associated locations showed up after “licenses and exceptions”.

The rephrasing drops “free and open source”, but while it's nice to drop that name, I think the wider “collaborative” covers that space with room to spare.